### ISSUE TITLE
Bag Of Holding disappeared

### ISSUE THREAD

**Issue by LaFesta1749** [adam9899/Issues#843]
So I had a "bag of holding" from the mod - "Extra Utilities 2" in which I store my precious stuff.

I opened it to get 1-2 black market containers (I had like 30 of them) and my bag closed and my Simple Storage Remote opened up. I was like "what the hell" so I closed it and looked for my bag in my inventory but it was gone. I looked on the ground - nothing. Looked inside the storage - There were 3 of them but all empty.

Basically it just disappeared into thin air.
